docker/push_image.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euxo pipefail
+
+docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
+docker push sifive/wit
```

Comment:
I think you need to set the tag/version here or else I believe it'll push `latest`.